2022Swerve/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,14 +26,18 @@ public class RobotContainer {
 
   private final Joystick driverJoystick = new Joystick(OIConstants.kDriverControllerPort);
 
+  private final SimulationSubsystem simSub = new SimulationSubsystem(swerveSubsystem);
+
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     swerveSubsystem.setDefaultCommand(new SwerveTeleopCommand(
       swerveSubsystem, 
       () -> -driverJoystick.getRawAxis(OIConstants.kDriverYAxis),
       () -> driverJoystick.getRawAxis(OIConstants.kDriverXAxis),
       () -> driverJoystick.getRawAxis(OIConstants.kDriverRotAxis)));
+      if (simSub == new SimulationSubsystem(swerveSubsystem)) {}
```

Comment:
What is the intention of this line?

It seems like you are constructing a new object here, instead of using it for the condition

Reply:
The simSub was declared as unused without it so we placed it there for the code to not be yellow. We have deleted this line.